clients/spring-boot-starter-camunda-sdk/src/main/java/io/camunda/zeebe/spring/client/properties/PropertyBasedZeebeWorkerValueCustomizer.java

```diff
@@ -117,10 +119,17 @@ private List<String> readVariablesAsTypeParameters(final MethodInfo methodInfo)
     parameters.forEach(
         pi ->
             ReflectionUtils.doWithFields(
-                pi.getParameterInfo().getType(), f -> result.add(f.getName())));
+                pi.getParameterInfo().getType(), f -> result.add(extractParameterName(f))));
     return result;
   }
 
+  private String extractParameterName(final Field field) {
```

Comment:
🔧 nitpicking: I'd have named this `extractFieldName`

Reply:
Sounds better, yes 👍